hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -325,6 +326,31 @@ public static SparkConf registerClasses(SparkConf conf) {
     }
   }
 
+  /**
+   * Deletes a bunch of keys from the Hoodie table, at the supplied commitTime
```

Comment:
Add proper Javadocs as this will become a public API

Reply:
I followed upsert doc ;) 
/** Upserts a bunch of new records into the Hoodie table, at the supplied commitTime */ 
Anyways, will fix it.